gadget/update_test.go

```diff
@@ -1032,18 +1041,22 @@ func (u *updateTestSuite) TestUpdateApplyUC20MissingInitialMapFullLogicOnlySyste
 	// system-boot volume can be updated as per policy since the initial mapping
 	// was missing
 
-	// mbr - bare structure
-	newData.Info.Volumes["pc"].Structure[0].Update.Edition = 1
-	// bios - partition w/o filesystem
-	newData.Info.Volumes["pc"].Structure[1].Update.Edition = 1
-	// ubuntu-seed
-	newData.Info.Volumes["pc"].Structure[2].Update.Edition = 1
-	// ubuntu-boot
-	newData.Info.Volumes["pc"].Structure[3].Update.Edition = 1
-	// ubuntu-save
-	newData.Info.Volumes["pc"].Structure[4].Update.Edition = 1
-	// ubuntu-data
-	newData.Info.Volumes["pc"].Structure[5].Update.Edition = 1
+	// mbr - bare structure, and bios - partition w/o filesystem
+	for i, imgName := range []string{"mbr.img", "bios.img"} {
+		imgPath := filepath.Join(newData.RootDir, imgName)
+		err = os.WriteFile(imgPath, nil, 0644)
+		c.Assert(err, IsNil)
+		newData.Info.Volumes["pc"].Structure[i].Content = []gadget.VolumeContent{{Image: imgName}}
+		newData.Info.Volumes["pc"].Structure[i].Update.Edition = 1
+	}
+	// ubuntu-{seed,boot,save,data}
+	for i, fName := range []string{"seed", "boot", "save", "data"} {
```

Comment:
I understand that the test was like this before but it's a bit weird to put content in save/data, same also in some other tests